RELEASE.txt

```diff
@@ -0,0 +1,25 @@
+
+This is the procedure for generating a new Dafny release.
+
```

Comment:
* Another step is to update the version number in dafny/Source/version.cs to be $VER. This should be done before step 3. The updated file should be pushed.
* Around step 5-6, one should also write some release notes.
* Github will ask for a name of the release. Use “Dafny $V”, where $V are the first three components of the 4-component $VER.

Reply:
OK